Store.js

```diff
@@ -140,14 +150,109 @@ define(["dcl/dcl", "decor/Invalidating"], function (dcl, Invalidating) {
 						collection.on("delete", this._itemRemoved.bind(this));
 						collection.on("refresh", this._refreshHandler.bind(this));
 					}
-					return this.processCollection(collection);
+				} else {
+					for (var i = 0; i < this.store.length; i++) {
+						// affect the callback to the observe function if the item is observable
+						Observable.observe(this.store[i], this.observeCallback.bind(this));
```

Comment:
What `Observable.observe()` returns should be kept somewhere so that it can be cleaned up as necessary (e.g. when the array item is removed from `this.store`, when the widget is being destroyed).